docs/Tomcat_Operation_Flow.md

```
@@ -0,0 +1,52 @@
+### 톰캣 서버 시작
+
+- Application.main()으로 톰캣 서버 프로그램 시작
+    - Tomcat tomcat 생성
+    - tomcat.start() 메서드 실행
+- tomcat.start()
+    - Connector connector 생성
+        - DEFAULT_PORT(8080), DEFAULT_ACCEPT_COUNT(100)으로 ServerSocket 생성
+        - 생성된 ServerSocket을 serverSocket 필드로 유지하고 있음
+    - connector.start() 메서드 실행
+- connector.start()
+    - tomcat 자신을 새로운 스레드로 실행 (Connectors는 Runnable을 상속받기에 스레드로 실행 가능)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)